CODE_REVIEW.md

```diff
@@ -0,0 +1,54 @@
+# Code Review
+
+1. The observable which is subscribed in `ngOnInit` is never unsubscribed which leads to the memory leak.
+
+2. FormateDate is impemented using a function. Function call triggers with every change detection which impacts application performance.
+
+3. Naming convention used for variables in `ngFor` are not readable.
+
+4. Testing cases failing in `reading-list.reducer.spec.ts`.
+
+5. Lifecycle hooks are added but never used.
+
+6. `a` tag is used for functionality of a button.
+
+# Code Improvements
+
+1. Added async pipe in `book-search.component.html` and removed the `ngOnInit` in `book-search.component.html`.
```

Comment:
Change it to:

Added async pipe in `book-search.component.html` and removed `ngOnInit` from `book-search.component.ts`.

Add in this statement why async is better. What it will do/help?